lib/matplotlib/colorbar.py

```diff
@@ -509,6 +510,66 @@ def __init__(self, ax, mappable=None, *, cmap=None,
         # Set the cla function to the cbar's method to override it
         self.ax.cla = self._cbar_cla
 
+    @property
+    def locator(self):
+        """
+        Major `~.Locator` being used for colorbar.
+        """
```

Comment:
```suggestion
        """The major tick `.Locator` of the colorbar."""
```
- If there is just one line, we put the quotes on the same line https://matplotlib.org/devdocs/devel/documenting_mpl.html#quote-positions
- `~` removes leading package/module names from links, but here you don't have any, so we can leave it out, which makes the code a bit more readable.
- "being used" does not add information -> can be left out.
Similar for all getters.
- I've added "*tick* Locator", while locators are a technical term and you can find out through the link, it may be worth associating all these properties with ticks explicitly.

Reply:
Went for `Major tick `.Locator` for the colorbar`